src/reshape.jl

```diff
@@ -61,5 +49,16 @@ function unstack(t::D, by = pkeynames(t); variable = :variable, value = :value)
     S = eltype(colnames(t))
     cols = S.(union(columns(t, variable)))
     T = eltype(columns(t, value))
-    unstack(D, T isa Type{<:DataValue} ? eltype(T) : T, pkeys(tgrp), columns(tgrp, value), cols)
+    unstack(D, Missing <: T ? nonmissing(T) : T, pkeys(tgrp), columns(tgrp, value), cols)
```

Comment:
```suggestion
    unstack(D, nonmissing(T), pkeys(tgrp), columns(tgrp, value), cols)
```
should be enough.

Reply:
Assuming you meant `Base.nonmissingtype`